docs/examples/bifacial/plot_bifi_model_mc.py

```diff
@@ -0,0 +1,96 @@
+"""
+Bifacial Modeling - modelchain
+==============================
+
+Example of bifacial modeling using the modelchain
+"""
+
+# %%
+# This example shows how to complete a bifacial modeling example using the
+# :py:class:`pvlib.modelchain.ModelChain` with the
+# :py:func:`pvlib.bifacial.pvfactors_timeseries` function to transpose
+# GHI data to both front and rear Plane of Array (POA) irradiance.
```

Comment:
Probably a good idea to explain why we have to mix procedural and ModelChain code here.  Maybe something like this?

    # Unfortunately ``ModelChain`` does not yet support bifacial simulation
    # directly so we have to do the bifacial irradiance simulation ourselves.
    # Once the combined front + rear irradiance is known, we can pass that
    # to ``ModelChain`` and proceed as usual.
    #
    # Future versions of pvlib may make it easier to do bifacial modeling
    # with ``ModelChain``.